test/watcher.test.js

```diff
@@ -0,0 +1,131 @@
+const path = require('path');
+const ncc = (global.coverage || true) ? require("../src/index") : require("../");
```

Comment:
What's the purpose of `global.coverage` here?

Reply:
This is the same thing we do in `test/index.test.js` which is that the coverage build uses the source files because the coverage maps only work against the original sources.